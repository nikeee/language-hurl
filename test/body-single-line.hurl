POST https://example.org/helloworld
`Hello world!`
HTTP 200
