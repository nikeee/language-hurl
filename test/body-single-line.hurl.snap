>POST https://example.org/helloworld
#^^^^ source.hurl keyword.control.method.hurl
#    ^ source.hurl
#     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hurl string.quoted.url.hurl string.quoted.url.hurl
>`Hello world!`
#^^^^^^^^^^^^^^ source.hurl string.quoted.backtick.hurl
>HTTP 200
#^^^^ source.hurl constant.language.version.hurl
#    ^^^^ source.hurl constant.numeric.status.hurl
>